controllers/comments.controllers.js

```diff
@@ -0,0 +1,19 @@
+const {selectCommentsByArticleId} = require("../models/comments.models")
+const {selectArticlesById} = require("../models/articles.models")
+
+const getCommentsByArticleId = (req, res, next) => {
+ const {article_id} = req.params
+ const pendingArticleById = selectArticlesById(article_id)
+ const pendingCommentsByFilmId = selectCommentsByArticleId(article_id)
+ Promise.all([pendingCommentsByFilmId, pendingArticleById]).then(([comments])=>{
+    res.status(200).send({comments})
```

Comment:
This looks like good code! If you bring back the test I've left another comment on, it should hopefully pass. If it doesn't, you're definitely on the right track!